lib/driver.js

```diff
@@ -366,7 +366,14 @@ class AndroidUiautomator2Driver extends BaseDriver {
 
     if (apiLevel >= 28) { // Android P
       this.log.info('Relaxing hidden api policy');
-      await this.adb.setHiddenApiPolicy('1', !!this.opts.ignoreHiddenApiPolicyError);
+      try {
+        await this.adb.setHiddenApiPolicy('1', !!this.opts.ignoreHiddenApiPolicyError);
+      } catch (err) {
+        this.log.errorAndThrow('Failed to disable hidden api policy. You could configure ' +
```

Comment:
Hidden APIs cannot be enabled. This might be happening because the device under test is not configured properly. See https://github.com/appium/appium/issues/13802  for more details. You could also set the 'ignoreHiddenApiPolicyError' capability to true in order to ignore this error, which might later lead to unexpected crashes of the automation server. Original error: ...